src/main/java/com/chessmates/lichess/data/LichessApi.groovy

```diff
@@ -11,15 +7,15 @@ interface LichessApi {
     /**
      * Get the players of a given Lichess team.
      */
-    LichessResultPage<Player> getPlayers(String teamId, int pageNumber, int pageSize)
+    def getPlayers(String teamId, int pageNumber, int pageSize)
 
     /**
      * Get games of a given player.
      */
-    LichessResultPage<Game> getGames(String playerId, int pageNumber, int pageSize)
+    def getGames(String playerId, int pageNumber, int pageSize)
```

Comment:
LichessResultPage as a wrapper is not a bad idea, this is a wrapper that contains multiple GamesJson I think. We could just abstract that GamesJson as an Object, but shape of the ResultPage could be defined?